controllers/async_controller.go

```diff
@@ -9,12 +9,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/Azure/azure-service-operator/pkg/resourcemanager"
-	"github.com/Azure/azure-service-operator/pkg/resourcemanager/config"
-	"github.com/Azure/azure-service-operator/pkg/secrets"
-	keyvaultsecretlib "github.com/Azure/azure-service-operator/pkg/secrets/keyvault"
-	telemetry "github.com/Azure/azure-service-operator/pkg/telemetry"
-	multierror "github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-multierror"
```

Comment:
Why use this vs `"k8s.io/apimachinery/pkg/util/errors"` which we've used elsewhere for error aggregation?

Reply:
This is V1 ASO, it used this package and not the other one we use for V2. Didn't make sense to change it since I'm just updating dependencies right now not trying to change functionality.